directjngine/src/test/java/com/softwarementors/extjs/djn/router/processor/standard/form/upload/UploadFormPostRequestProcessorTest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright © 2008, 2012 Pedro Agulló Soliveres.
```

Comment:
Since this is a new file I believe the copyright of this file belongs to us

Reply:
Fair enough, I wasn’t exactly sure how to handle that ;)

Reply:
https://github.com/sonatype/directjngine/pull/11/commits/c6a874bee6a6aedca631794d83c7a51e4b7b0664